includes/admin/class.llms.admin.setup.wizard.php

```diff
@@ -25,331 +25,73 @@
  *              - `LLMS_Admin_Setup_Wizard::scripts()` method
  *              - `LLMS_Admin_Setup_Wizard::watch_course_generation()` method
  */
-class LLMS_Admin_Setup_Wizard {
-
-	/**
-	 * Instance of WP_Error
-	 *
-	 * @var WP_Error
-	 */
-	public $error;
+class LLMS_Admin_Setup_Wizard extends LLMS_Abstract_Admin_Wizard {
 
 	/**
-	 * Constructor
+	 * Configure wizard.
 	 *
-	 * @since 3.0.0
-	 * @since 4.4.4 Remove output of inline scripts.
+	 * @since [version]
```

Comment:
We're keeping the `__construct` method, so we need to keep to changelog and add a new one, right?